src/solutions/concepts/kafka-proxies/secure-private-access.md

```diff
@@ -19,19 +18,30 @@ This setup establishes a fully private, secure, and scalable communication chann
 - **Custom Wildcard DNS** & Route 53 Hosted Zone Integration.
 - **Unified Domain Name** for Kafka clients, streamlining configuration.
 - **Eliminates** the need to **manually whitelist** each bootstrap endpoint to enable access.
 - Seamless end-to-end **TLS** handshake.
-- **Auto-Scaling** <ZillaPlus/> Instances
+- **Auto-Scaling** <ZillaPlus/> Instances.
+- Deployed behind a **Network Load Balancer** for high availability and efficient request routing.
 
-You will need to choose a wildcard DNS pattern to use for intranet access to the brokers in your Kafka cluster. These wildcard DNS names must resolve to the IP address(es) where the <ZillaPlus/> proxy is deployed. The <ZillaPlus/> proxy must also be configured with a TLS server certificate representing the same wildcard DNS pattern.
+### Many-to-One Private Access
 
-::: info
-Aklivity recommend using the format `*.<region>.<custom-domain>` for your custom wildcard DNS domain. This removes the need for additional client configuration, as the MSK Serverless cluster region can be directly inferred from the custom domain itself.
-:::
+Multiple Kafka clients from cross-account VPCs securely connect to a single Amazon MSK Serverless cluster. This approach simplifies multi-tenant access and ensures a unified, private connectivity model.
 
-## Deploy with Terraform
+![Many to One Private Access Overview](/many_to_one.png)
 
-Follow the [Secure Private Access with Terraform](https://github.com/aklivity/zilla-plus-aws-templates/tree/main/amazon-msk/cdktf/secure-private-access) guide to generated or deploy a custom Terraform template using [CDKTF](https://developer.hashicorp.com/terraform/cdktf). This Terraform script can be configured to deploy `SASL/SCRAM authentication`, `Mutual TLS (mTLS) authentication` or `Unauthorized access` to setup connectivity to your MSK Serverless cluster with a wildcard DNS pattern.
+### One-to-Many Private Access
+
+Enables Kafka client to securely access multiple Amazon MSK Serverless clusters deployed across different VPCs.
+
+![One to Many Private Access Overview](/one_to_many.png)
+
+You will need to choose a wildcard DNS pattern to use for intranet access to the brokers in your Kafka cluster. These wildcard DNS names must resolve to the IP address of the VPC Endpoint in the client VPC, which then routes traffic via the VPC Endpoint Service to the ZillaPlus Network Load Balancer (NLB).
+
+Additionally, the <ZillaPlus/> proxy must also be configured with a TLS server certificate representing the same wildcard DNS pattern.
+
+## Deploy with CDK
+
+Follow the [Secure Private Access with CDK](https://github.com/aklivity/zilla-plus-aws-templates/tree/main/amazon-msk/cdk/secure-private-access) guide to generated or deploy a custom AWS CDK stack. This CDK-based deployment supports `SASL/SCRAM authentication`, `Mutual TLS (mTLS) authentication` or `Unauthorized access` to setup connectivity to your MSK Serverless cluster with a wildcard DNS pattern.
```

Comment:
```suggestion
Follow the [Secure Private Access with CDK](https://github.com/aklivity/zilla-plus-aws-templates/tree/main/amazon-msk/cdk/secure-private-access) guide to generate or deploy a custom AWS CDK stack.
```
Only IAM authorization is relevant for MSK Serverless clusters, there are no choices like MSK Provisioned clusters.